web/src/SidebarItemView.tsx

```diff
@@ -38,6 +38,11 @@ export const SidebarItemRoot = styled.li`
   ${StarResourceButtonRoot} {
     margin-right: ${SizeUnit(1.0 / 12)};
   }
+
+  /* groupIndent is used for un-grouped items in the grouped sidebar view */
+  &.groupIndent {
```

Comment:
I think I understand what the code is doing but I'm finding myself confused about what "group" means.
1. By itself, "groupIndent is used for un-grouped items" reads pretty strangely. I guess the "group" there is referring to the view rather than the items, but it took me a while to get there. Maybe "groupedSidebarResourceIndent"?
2. The comment here says it's used for ungrouped items, but [below](https://github.com/tilt-dev/tilt/pull/4765/files#diff-2114c24a4cdf4a64c9f1ce43f4a80ab57f605472e53a40564b0f2dccf7f44acfR275) it seems to only be used when `inGroup` is true, which seems like the opposite of what the comment says?
3. But then [further down](https://github.com/tilt-dev/tilt/pull/4765/files#diff-363468d2eab0757279a1799e1c908efe2d100ba43fce7e10c34e593046610c2aR220), it turns out `inGroup` is only true for the Tiltfile, which I would have thought was the only resource not in a group.

Reply:
that's totally fair 😅 i found myself struggling a bit with the naming. this class is more like "indent un-grouped items to align with grouped items in grouped view". what do you think about the `inGroup` option prop becoming `groupView`? i can update the comment with a clearer explanation.

Reply:
i improved the comment and naming, so hopefully it's more clear what it does!